### ISSUE TITLE
Type error in v13.1.1

### ISSUE THREAD

**Issue by evanjo03** [twitter/twemoji#534]
**Type error when using version 13.1.1**

Appears to the same problem described in [this](https://github.com/twitter/twemoji/issues/532) issue.  Looks like a [fix](https://github.com/twitter/twemoji/pull/533) was applied for v14, but not for v13.

```
node_modules/twemoji/index.d.ts:32:1 - error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.

32 const twemoji: {
   ~~~~~


Found 1 error in node_modules/twemoji/index.d.ts:32
```

**To Reproduce**
1. `npm init @open-wc` and follow prompts
2. `cd [project-name] && npm i`
3. `npm i @joeattardi/emoji-button`
4. Add `import { EmojiButton } from "@joeattardi/emoji-button";`
5. `npm run tsc`
6. See error

**Expected behavior**
Tsc occurs without issues.

**Screenshots**
![image](https://user-images.githubusercontent.com/42701217/158400054-91f8f89b-1180-4575-8654-94091dace4d7.png)

**Environment**
Chrome 99.0.4844.51

**Comment by hthetiot** [comment]
Not exactly the same fix since 13.x expose ParseObject and possibly other module exports, my suggestion migrate to 14.0.1 and use `import twemoji from 'twemoji';`

**Comment by hthetiot** [comment]
@evanjo03 once https://github.com/twitter/twemoji/pull/535 is merged we can make PR to fix @joeattardi/emoji-button in the meantime adding `npm i twemoji@13.0.0` after the install fix the issue.

Workarround:

1. npm init @open-wc and follow prompts
2. cd [project-name] && npm i
3. npm i @joeattardi/emoji-button
3. Add import { EmojiButton } from "@joeattardi/emoji-button";
4. npm i twemoji@13.0.0
5. npm run build

**Comment by jdecked** [comment]
@hthetiot @evanjo03 this should be fixed in v14.0.2. Can you upgrade and LMK if you're still having issues?
